packages/features/schedules/lib/use-schedule/useSchedule.ts

```diff
@@ -29,24 +29,31 @@ export const useSchedule = ({
   return trpc.viewer.public.slots.getSchedule.useQuery(
     {
       usernameList: getUsernameList(username ?? ""),
-      eventTypeSlug: eventSlug!,
+      // Prioritize slug over id, since slug is the first value we get available.
```

Comment:
Non dynamic events need eventTypeId only
<img width="1030" alt="Screenshot 2023-08-07 at 10 03 48 AM" src="https://github.com/calcom/cal.com/assets/1780212/cbdc9430-1546-4bca-a390-cb5446ae3850">

Reply:
This still pending as I think test related to dynamic should fail here.

Reply:
![image](https://github.com/calcom/cal.com/assets/1780212/61db6bd9-bbfd-4a5e-9cd3-1e1ae665c5f7)

Regular events(Non Dynamic) aren't showing availability if they don't have slug 15min,60min,30min. 
This is due to `getSchedule` not receiving eventTypeId and the above logic considering the request as dynamic booking availability request.

Reply:
I couldn't reproduce it. Mine availability is working fine here. Can you give me exact steps to do so?
![image](https://github.com/calcom/cal.com/assets/4461358/ed3b813b-4761-4e5c-801a-e5b07ec9ece5)

Reply:
That is super weird. Giving you a loom

Reply:
Here it is https://www.loom.com/share/058304b8913d4a38be70b73c6af88acc